ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -6,13 +6,13 @@
 {-# LANGUAGE UndecidableSuperClasses #-}
 module Ouroboros.Consensus.Shelley.Eras (
     -- * Eras based on the Shelley ledger
-    ShelleyEra
```

Comment:
Ah, OK. Here's an example where it was intentionally out-of-order. What are our options, @EncodePanda? For example are there any easy options for special-casing this rule for the era names? They have a meaningful non-alphabetical chronological order which is what we have been using throughout.

Reply:
I'm afraid there is no way to work around this at the moment. Possible options:

1. discard this file from being formatted completely OR format this file with a modified configuration
This would work but the approach does not scale. We would have to explicitly exclude files or keep a list of which files we want to format with a specific set of rules

2. turn off the `sort` option in the module header.
This means that the `module_header` step would stop completely sorting the export lists.  It would only check that: each export is in a separate line, open bracket starts on the same line as module name, indentation is correct (according to Style Guide).

3. Don't bother that the exported eras are not in order in which the `Cardano` combines them 
I honestly wouldn't mind. If anything I prefer consistency.

Reply:
We decided to sort them alphabetically, not chronologically. We think the extra automation is worth it, and moreover is consistent with how eg imports will be sorted.

If `stylish-haskell` were to offer an extensible ordering that the config could customize, then we might consider reintroducing the historical sequence of era names as part of the ordering.